pages/pipelines/wait_step.md

```diff
@@ -106,7 +106,9 @@ steps:
 
 <%= image "continue-on-fail-example2.png", width: 1402/2, height: 417/2, alt: "Screenshot of a failed pipeline which did run the continue_on_failure wait steps" %>
 
-The explicit null `~` character used in the above examples isn't required, but is recommended as a best practice. It ensures that nothing else is accidentally added to the `wait` before the `continue_on_failure` attribute.
+The explicit null `~` character used in the above examples isn't required, but is recommended as a best practice. It ensures that nothing else is accidentally added to the `wait` before the `continue_on_failure` attribute. 
+
+If command step before wait step with `continue_on_failure` is cancelled then all the steps after this cancelled step will not get executed because cancellation is purposefully excluded from continue_on_failure behaviour. In above example, if we cancel `echo THIRD command` which is step before wait step with `continue_on_failure` then `echo FOURTH command` step will not get executed.
```

Comment:
This is a bit of a topic shift after the other content in this section. I think it might benefit from including a heading to help encapsulate it. Thinking:

````suggestion
### Continuing after cancelation

The `continue_on_failure` attribute enables builds to continue after a failed job but not after a cancelation. If you cancel a job, any subsequent wait steps with `continue_on_failure: true` do not execute.

For example, if you cancel the first command, the second command doesn't run in the following `plugin.yml` file:

```yml
steps:
  - command: "run-first-command.sh"
  - wait: ~
    continue_on_failure: true
  - command: "run-second-command.sh"
```
````

What do you think?

Reply:
Yes, this simplifies it. Thank you for the suggestion